### ISSUE TITLE
Access-Control-Allow-Headers

### ISSUE THREAD

**Issue by Jestric-sys** [bluenviron/mediamtx#2018]
## Describe the feature

Add header "Authorization" in file webrtc_http_server.go

```go
ctx.Writer.Header().Set("Access-Control-Allow-Headers", "Content-Type, If-Match, Authorization")
```

Because I can't send basic authorization. Error CORS Access-Control-Allow-Headers.

**Comment by aler9** [comment]
Thanks for reporting the issue, this is fixed by #2020.

**Comment by aler9** [comment]
added in v0.23.8
